pf.go

```diff
@@ -1,5 +1,3 @@
-// +build openbsd freebsd
```

Comment:
is it intentional ?

Reply:
yes. So at least the CI can verify that it builds and run tests